### PR TITLE
Make radio and checkbox labels inline-block

### LINKED ISSUE #543: Radios labels are full width - should be the width of the label text?
Radio labels extend to the full width of the container. This means the clicks apparently in space on the righthand side activate radios.

This was not the case with Elements where labels only extended as far as the text.

### PR DESCRIPTION
**Pull request by igloosi**
This fixes #543 regression, reintroduced with conditional reveal.

### CODE REVIEW

**Comment by 36degrees** [comment]
@nickcolley any idea why the labels were made `display: block`? Was there an issue in a particular browser we need to test?

**Comment by igloosi** [comment]
IE's seem to be fine with inline-block
ie8
ie9
ie10
ie11

**Comment by nickcolley** [comment]
Only thing I can think of is potential wrapping, with the conditional element, but maybe I just made a mistake.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,11 +22,8 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
-
-  Description goes here (optional)
-
-  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
+- Make radio and checkbox labels `inline-block` instead of `block`
```
Comment (nickcolley) [inline CHANGELOG.md]: Can we reword this to explain what it fixes?

"Fixes radio and checkbox labels extending full width of page"
